dotCMS/src/main/java/com/dotcms/util/ReturnableDelegate.java

```diff
@@ -11,5 +11,5 @@
      *
      * @return Object the information returned
      */
-    public T execute();
+    public T execute() throws Throwable;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)